### PR TITLE
[keymap] plattfot - Keymap updates for Kyria

### PR DESCRIPTION
**Pull request by plattfot**
## Description
- Major change in the keymap to work with EurKey. Which relaxes some
  constraints it had before when it had to take in consideration two
  layouts.

  With this the parenthesis can be moved to a better location instead
  of being in the top right corner.

  This also allows `esc`, `del` and `rctrl` to be moved to the base layer.

  Only downside is that `ctrl+lalt` needed to be removed and instead
  `AltGr` takes it's place. Add `rctrl` on right thumb cluster to
  compensate for this which need some reorganization on the thumb
  cluster.

- Split the symbol and function keys layer into two layers, one for
  each hand. Make it easier to press symbols and function keys.

- Add some symbols specific for the EurKey layout.

- Change from running `C-<tab>` `S-C-<tab>` to `page up`/`down` for the right
  rotary. As holding `ctrl` and using `page up`/`down` works the same in
  firefox. Which allows the rotary to be useful for other things.

- Move scroll lock and insert to right rotary.

- Introducing close tap (`CLO_TAP`), which is a combination of the
  double tap feature and my macros. E.g. pressing `CLO_TAP` and `(` will
  generate `()←`. Which removes the need of the macros and makes it more
  useful than `DBL_TAP` as it now saves me some keypresses. `CLO_TAP` exist
  on both the left and right hand layers to make it easy to use.

- Use text for the secondary oled, firmware is too big after rebasing
  on upstream master.

- Update image in the readme to reflect my new layout.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/kyria/keymaps/plattfot/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # plattfot's keymap for [Kyria](https://github.com/splitkb/kyria)
 
-![Keyboard](https://i.imgur.com/Pvsm973l.jpg)
+![Keyboard](https://i.imgur.com/mcefoHn.jpg)
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/README.md:3]:
```suggestion
![Keyboard](https://i.imgur.com/mcefoHnh.jpg)
```

File: keyboards/kyria/keymaps/plattfot/keymap.c [hunk 2]
```diff
@@ -18,19 +18,26 @@
...
-static bool double_tap_it = false;
+// Will insert the closing equivalent key and move the cursor inside.
+// For example clo_tap and then ( will result in (|), where | is the
+// cursor. For " it will be "|" as the close equivalent key is the
+// same key.
+enum close_tap_modes {
+  CLO_DISABLED = 0x000,
+  CLO_PRESSED  = 0x001, // Close tap key is pressed
+  CLO_ACTIVE   = 0x010, // Close tap next keypress
+  CLO_USED     = 0x100, // Turn off when close tap key is released
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/keymap.c:35]: Why not:
```suggestion
  CLO_DISABLED = 0b000,
  CLO_PRESSED  = 0b001, // Close tap key is pressed
  CLO_ACTIVE   = 0b010, // Close tap next keypress
  CLO_USED     = 0b100, // Turn off when close tap key is released
```
?

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kyria/keymaps/plattfot/keymap.c (hunk 2 above)
Comment (plattfot) [reply keyboards/kyria/keymaps/plattfot/keymap.c]: I didn't know C supported `0b` prefix. Thanks for pointing that out. Update to using that in commit 2538722.

File: keyboards/kyria/keymaps/plattfot/README.md (hunk 1 above)
Comment (plattfot) [reply keyboards/kyria/keymaps/plattfot/README.md]: Updated in  2548722.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**